vllm/model_executor/sampling_metadata.py

```diff
@@ -236,4 +318,40 @@ def from_lists(cls, temperatures: List[float], top_ps: List[float],
                                                            non_blocking=True),
             prompt_tokens=prompt_tensor.to(device=device, non_blocking=True),
             output_tokens=output_tensor.to(device=device, non_blocking=True),
+            sampling_seeds=sampling_seeds_gpu,
+            sample_indices=sample_indices_t.to(device=device,
+                                               non_blocking=True),
+            extra_seeds=extra_seeds_gpu,
         )
+
+    @staticmethod
+    def _get_sequence_seeds(
+        seed: int,
+        *extra_entropy: int,
+        seeds_to_generate: int,
+        is_greedy: bool,
+    ):
+        """Get `seeds_to_generate` child seeds from `seed` and extra entropy."""
+        # TODO(yard1): make sure sequences in the same group with
+        # best_of > 1 have different seeds (not trivial...)
```

Comment:
What's your thought on keeping a simple counter for sequence group and do the following in `from_sampling_metadata`
```
seq_idx = 0
for seq_id in seq_ids:
...
    extra_entropy = extra_entropy + (seq_idx, ) if extra_entropy else (seq_idx, )
    seq_idx += 1
...
```

Reply:
I think I will just use the seq id as entropy. Though one thing I am worried about is whether the seq id will be deterministic should the request be submitted again. I _think_ it should be, but I wasn't sure, which is why I left it as todo here. Will change it though.